xml/System/Math.xml

```diff
@@ -6137,32 +6138,35 @@ The following example demonstrates how to use the <xref:System.Math.Min%2A> meth
 | 1                              | Any value except `NaN`                                    | 1                              |
 | 1                              | `NaN`                                                     | 1 (`NaN` on .NET Framework)    |
 | Any value except 1             | `NaN`                                                     | `NaN`                          |
-| `NegativeInfinity`             | < 0                                                       | 0                              |
-| `NegativeInfinity`             | Positive odd integer                                      | `NegativeInfinity`             |
-| `NegativeInfinity`             | Positive but not an odd integer                           | `PositiveInfinity`             |
-| < 0 but not `NegativeInfinity` | Not an integer, `NegativeInfinity`, or `PositiveInfinity` | `NaN`                          |
-| -1                             | = `NegativeInfinity` or `PositiveInfinity`                | `NaN`                          |
+| `NegativeInfinity`             | < 0 and finite and odd integer                            | -0                             |
```

Comment:
I'm not sure if it makes sense to mark the 0 return values as +, -, or ± (as in `IEEE 754 2019`).

Reply:
I think we need to keep this information, as it is an important detail of what the function returns. `+0` and `-0` are different values for floating-point and can cause a large difference in results when input to other algorithms. A user seeing simply `0` may expect it to be positive and that can cause them to do the wrong things when the actual result would be `-0`